config/quality.gradle

```diff
@@ -105,3 +105,8 @@ android {
         xmlOutput file("$reportsDir/lint/lint-result.xml")
     }
 }
+
```

Comment:
Looks good, but don't add this piece of code since it has been merged into master.

Reply:
yes sure